aten/src/ATen/native/cuda/Embedding.cu

```diff
@@ -327,7 +327,7 @@ Tensor embedding_dense_backward_cuda(const Tensor & grad_, const Tensor & indice
       //  count: 1 3 3 3 2 2 1 2 2
       cuda::cub::inclusive_scan_by_key(
         thrust::make_reverse_iterator(sorted_data + num_indices),
-        thrust::make_reverse_iterator(count_data + num_indices),
+        thrust::make_reverse_iterator(static_cast<const index_t*>(count_data) + num_indices),
```

Comment:
How come you need this cast

Reply:
It doesn't need it, i just thought it made it more clear that one was an input and one was an output. Undo?